src/main/java/org/fcrepo/spec/testsuite/versioning/AbstractVersioningTest.java

```diff
@@ -79,6 +88,20 @@ protected boolean hasHeaderValueInMultiValueHeader(final String headerName, fina
 
     }
 
+    protected void confirmPresenceOfVersionLocationHeader(final Response response) {
+        final String location = response.getHeader("Location");
+        Assert.assertTrue(location.contains("fcr:versions"));
+    }
+
+    protected void confirmPresenceOfLinkValue(final String linkValue, final Response response) {
```

Comment:
I missed that this got moved to AbstractTest.java - I'll remove this function soon.